src/cgame/etj_cgaz.cpp

```diff
@@ -429,8 +429,6 @@ float CGaz::getOptAngle(const playerState_t &ps, pmove_t *pm, bool alternate) {
     wishspeed = static_cast<float>(ps.speed) * ps.sprintSpeedScale;
   }
 
-  UpdateDraw(wishspeed, &ps, pm);
-
   // no meaningful value if speed lower than ground speed or no user input
   if (state.vf < state.wishspeed || (cmd.forwardmove == 0 && cmd.rightmove == 0)) {
```

Comment:
Removing this, while good since this unnecessarily updates the entire CGaz state, now breaks anything that uses `getOptAngle` as it relies on `state.vf/wishspeed` which are not calculated.

Reply:
Fixed now. added this back and moved updatesnap out of updatedraw method. :)